tools/add_triage_signature.py

```diff
@@ -594,11 +594,18 @@ def _prepare_output(cls, output):
         if len(output) == 0:
             return output
 
-        if len(output) > cls.TRUNCATE_OUTPUT_LENGTH:
-            half = cls.TRUNCATE_OUTPUT_LENGTH // 2
-            output = f"{output[:half]} ... **OUTPUT HAS BEEN TRUNCATED** ... {output[-half:]}"
+        pre = "{code:borderStyle=none|bgColor=transparent} "
+        post = " {code}"
+        output_list = output.replace("|", "¦").split("\\n")
 
-        return "{code:none}%s{code}" % (output.replace("|", "¦"))
+        if len(output_list) > cls.TRUNCATE_OUTPUT_LINES:
+            half = cls.TRUNCATE_OUTPUT_LINES // 2
+            output_list = output_list[:half] + ["**OUTPUT HAS BEEN TRUNCATED**"] + output_list[-half:]
+
+        output_list = list(filter(lambda x: x != "\\\"", output_list))
+        output_list_wrapped = [pre + sub + post for sub in output_list]
```

Comment:
what ``sub`` means? maybe rename

Reply:
sublist :) I'll change to 'line', maybe clearer.